include/verilated_vpi.cpp

```diff
@@ -2447,6 +2550,17 @@ vpiHandle vpi_put_value(vpiHandle object, p_vpi_value valuep, p_vpi_time /*time_
         return nullptr;
     }
     if (const VerilatedVpioVar* const vop = VerilatedVpioVar::castp(object)) {
+        if (flags & vpiInertialDelay) {
+            if (!VerilatedVpiPutHolder::canInertialDelay(valuep)) {
+                VL_VPI_ERROR_(
+                    __FILE__, __LINE__,
+                    "%s: Unsupported p_vpi_value as requested for %s with vpiInertialDelay",
```

Comment:
```suggestion
                    "%s: Unsupported p_vpi_value as requested for '%s' with vpiInertialDelay",
```

Also will need a test to cover this case.

Reply:
> Also will need a test to cover this case.

Do you have any guidance for what should be a `VL_VPI_WARNING_` vs `VL_VPI_ERROR_`?  I changed this to a warning which made adding checks to `t_vpi_var` easy since it doesn't cause the process to abort.  However, now I'm not sure which way is correct.  E.g. supplying an unsupported format is an error, but an unsupported object type is a warning.

Also, if I go back to an error, I believe I'll need to make a bunch of `_bad` tests (of which there are currently none for VPI as far as I can tell) since each incorrect call will kill the process.

Reply:
I generally use warning, as it seems the VPI was assuming that in most cases, so I'm good with your choice.